docs/install.md

```diff
@@ -0,0 +1,151 @@
+# Install
+
+## Manual deploy
+
+To manually install the operator (on all its dependant resources) on default
+namespace `bank-vaults-helm-operator-system` without using OLM, you can use
+the following make target:
+
+```bash
+make deploy
+```
+
+* Then create any [OperatorConfig resource type](../config/samples/operator_v1alpha1_operatorconfig.yaml).
+
+```yaml
+apiVersion: operator.vault.banzaicloud.com/v1alpha1
+kind: OperatorConfig
+metadata:
+  name: example
+spec:
+  image:
+    bankVaultsRepository: ghcr.io/banzaicloud/bank-vaults
+    repository: ghcr.io/banzaicloud/vault-operator
+  resources:
+    requests:
+      cpu: 100m
+      memory: 200Mi
+    limits:
+      cpu: 100m
+      memory: 400Mi
+```
+
+* Once tested, delete created operator resources using the following make target:
+
+```bash
+make undeploy
+```
+
+## OLM manual deploy
+
+If you want to install a specific version of the operator **manually** via OLM
+
+* Deploy with `operator-sdk` using the following command:
+
+```bash
+operator-sdk run bundle quay.io/3scale/bank-vaults-helm-operator:vv1.15.6
```

Comment:
Double `vv`?

Reply:
Fixed in https://github.com/3scale-ops/bank-vaults-helm-operator/pull/1/commits/00aa57c78dadaf16b7d85eb47f1a1c64b0e72065